model-optimizer/install_prerequisites/install_prerequisites.bat

```diff
@@ -35,10 +37,16 @@ if not "%python_ver%"=="okay" (
 
 :: install Python modules
 
+set USE_VENV="false"
+set VENV_DIR=%USERPROFILE%\Documents\Intel\OpenVINO\venv_openvino
```

Comment:
Is this directory created after OpenVINO installation?

Reply:
No, but it also will be created using validation scripts from `deployment_tools/demo` folder, so we can reuse it.